.rubocop.yml

```diff
@@ -46,7 +46,7 @@ Layout/HashAlignment:
 
 # To be tweaked until we find the right balance
 Metrics/MethodLength:
-  Max: 12
+  Max: 14
```

Comment:
For CaseDecisionTree

Reply:
I think we can revert this change because in a previous PR by @willmcb he already opted out the case decision tree from the offending cop with `rubocop:disable Metrics/MethodLength`

Reply:
We had disabled for the destination method before, on an earlier PR, and the discussion was that we would tweak the method length rather than disabling it. Are we reverting that decision then?

Reply:
Yes, we want to keep a sensible max method length. But decision trees are going to always be lengthy by definition so it doesn't make sense to keep increasing this cop max every time a decision tree grows in size.

So for decision trees that go over the max (in this case 12 lines) like the case decision tree, we opt out that method (`#destination`) in the code itself.

It allow us to still keep a sensible max value in other places and other methods.

If we keep finding ourselves going over the limit (12 right now) in other methods and places that are not just decision trees, then it may make sense to increase the limit instead of making the methods more difficult to read just to save lines of code.

Hope it makes sense 😄 We can discuss on a slack call maybe that's easier.

Reply:
No that's fine, that makes sense. I think I just misunderstood when we discussed last time.